src/app/shared/footer/footer.component.pug

```diff
@@ -72,8 +72,8 @@
             #google_translate_element.translate-flyout
         .col-md-3.col-6
           div.social-icons
-            a.icon.icon-twitter(target="_blank" href="http://twitter.com/#!/artstor" title="Follow us on Twitter")
-            a.icon.icon-facebook(target="_blank" href="https://www.facebook.com/ARTstor" title="Keep up with Artstor on Facebook")
+            a.icon.icon-twitter(target="_blank" href="http://twitter.com/#!/artstor" aria-label="Follow us on Twitter")
+            a.icon.icon-facebook(target="_blank" href="https://www.facebook.com/ARTstor" aria-label="Keep up with Artstor on Facebook")
```

Comment:
```suggestion
            a.icon.icon-facebook(target="_blank" href="https://www.facebook.com/ARTstor" aria-label="Keep up with Artstor on Facebook.")
```

Reply:
Changes made in the next commit `4d90ebb`.